packages/sbom-tools/src/webpack-dependencies-plugin.ts

```diff
@@ -69,9 +71,44 @@ export class WebpackDependenciesPlugin implements WebpackPluginInstance {
         this.resolvedModules.add(packageJsonPath);
       }
     }
-  };
+  }
 
   apply(compiler: Compiler): void {
+    const webpackModulePath = this.getWebpackModulePath(new Error());
+    console.log(webpackModulePath);
+
+    this.addIncludedPackages(compiler);
+
+    compiler.hooks.done.tapAsync(PLUGIN_NAME, (stats, done) => {
+      const { modules } = stats.toJson();
+
+      console.log(modules?.filter((m) => m.moduleType === 'runtime'));
+
+      modules?.forEach(({ type, nameForCondition }) => {
+        if (
+          type === 'module' &&
+          nameForCondition &&
+          this.isThirdPartyModule(nameForCondition)
+        ) {
+          console.log(nameForCondition);
+
+          this.resolvedModules.add(nameForCondition);
+        }
+      });
+
+      if (
+        modules?.find(
+          (m) =>
+            m.moduleType === 'runtime' && m.name?.startsWith('webpack/runtime')
+        ) &&
+        webpackModulePath
+      ) {
+        this.resolvedModules.add(webpackModulePath);
+      }
+
+      done();
+    });
+
     compiler.hooks.shutdown.tapPromise(PLUGIN_NAME, async () => {
```

Comment:
Doesn't make that much of a difference, but this probably can be just done in the same stats hook? I wonder if you kept it in this one for some particular reason 🤔

Reply:
Multi-config, right, makes sense!